### PR TITLE
[CHORE] Deployment: docker compose with local auth and install script adjustments

### PR DESCRIPTION
**Pull request by mtmr0x**

### CODE REVIEW

File: deploy/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-    volumes:
-      - hoopdata:/hoopdata
-    env_file: ".env"
-    command: bootstrap.sh
   db:
     image: postgres:16
     networks:
       - hoopdev
+    volumes:
+      - hoopdata:/hoopdata
```
Comment (sandromello) [inline deploy/docker-compose/docker-compose.yml]: There's no need for this

File: deploy/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -120,17 +50,15 @@ services:
...
     networks:
       - hoopdev
     environment:
-      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/hoop/bin
       - POSTGRES_DB_URI=postgres://postgres:postgres@db:5432/hoopdb?sslmode=disable
       - LOG_LEVEL=info
       - LOG_ENCODING=console
-      - HOOP_PUBLIC_HOSTNAME=${HOOP_PUBLIC_HOSTNAME}
-      - HOOP_TLS_MODE=${HOOP_TLS_MODE}
     depends_on:
```
Comment (sandromello) [inline deploy/docker-compose/docker-compose.yml:60]: Remove the `volumes` attribute if you're not using it.

File: deploy/docker-compose/.env-sample [hunk 3]
```diff
@@ -1,7 +1,9 @@
-# your instance public hostname
-HOOP_PUBLIC_HOSTNAME=
-# enabled|disabled
-HOOP_TLS_MODE=
+# In case you are running the application locally without a custom Identity Provider,
+# you need to provide the JWT_SECRET_KEY value in the .env file.
+# Otherwise, just leave it empty and configure the IDP as you will find
+# below. Any issues, you can check our documentation at https://hoop.dev/docs
+# JWT secret key
```
Comment (sandromello) [inline deploy/docker-compose/.env-sample]: Add instructions to generate a secure random string for this value.
```sh
openssl rand 128 | base64
```

File: deploy/docker-compose/.env-sample [hunk 4]
```diff
@@ -1,7 +1,9 @@
-# your instance public hostname
-HOOP_PUBLIC_HOSTNAME=
-# enabled|disabled
-HOOP_TLS_MODE=
+# In case you are running the application locally without a custom Identity Provider,
```
Comment (sandromello) [inline deploy/docker-compose/.env-sample:1]: Did you test it with a custom identity provider to see if it works?

Comment (mtmr0x) [reply deploy/docker-compose/.env-sample:1]: works like a charm!

**Review by sandromello — APPROVED** [review]

**Pull request merged**